### PR TITLE
Refactor/namespace game

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by m4rtint** [comment]
[Refactor Games Folder](https://trello.com/c/8ry12iBy/436-refactor-games-folder)

File: Assets/_PartyParkade/Prefabs/Player/Player.prefab [hunk 1]
```diff
@@ -603,12 +603,22 @@ PrefabInstance:
     - target: {fileID: 1872667390450804921, guid: 656c4aa2e60b7489bae3472e003c8ac9,
         type: 3}
       propertyPath: m_fontSize
-      value: 74.15
+      value: 50
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Prefabs/Player/Player.prefab]: Any reason why the player prefab was edited?

File: Assets/_PartyParkade/Script/Game/Car/CameraShake.cs [hunk 2]
```diff
@@ -8,75 +8,78 @@
 using Cinemachine;
 using UnityEngine;
 
-/// <summary>
-/// Camera Shaking behaviour
-/// </summary>
-public class CameraShake : MonoBehaviour
+namespace _PartyParkade.Script.Game.Car
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Script/Game/Car/CameraShake.cs]: I just saw this in the previous PR so which namespace are we sticking with?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/Car/CameraShake.cs]: The name space depends on which folder they are in

File: Assets/_PartyParkade/Prefabs/Player/Player.prefab (hunk 1 above)
Comment (m4rtint) [reply Assets/_PartyParkade/Prefabs/Player/Player.prefab]: The serialized value was renamed

Comment (m4rtint) [reply Assets/_PartyParkade/Prefabs/Player/Player.prefab]: So those renamed serialized values got reset

File: Assets/_PartyParkade/Script/Game/DebugInventory.cs [hunk 3]
```diff
@@ -3,70 +3,74 @@
...
+            {
+                {KeyCode.Alpha1, POWERUPS.BouncyObject},
+                {KeyCode.Alpha2, POWERUPS.GravityWell},
+                {KeyCode.Alpha3, POWERUPS.OilSpill},
+                {KeyCode.Alpha4, POWERUPS.SwitchPlayers},
+                {KeyCode.Alpha5, POWERUPS.Boost},
+                {KeyCode.Alpha6, POWERUPS.SwitchAllPlayers},
+                {KeyCode.Alpha7, POWERUPS.SwitchTwoPlayersRandomly}
+            };
+            foreach (KeyValuePair<KeyCode, POWERUPS> entry in powerUpKeyMap)
```
Comment (wbeltran777) [inline Assets/_PartyParkade/Script/Game/DebugInventory.cs]: Should we be iterating over every item? Also maybe we should cache the dictionary

**Pull request merged**